style.css

```diff
@@ -0,0 +1,113 @@
+
+/*General Page Style */
+*{
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+	font-family: sans-serif;
+}
+
+/*Profile Section at Home Page Style */
+.main_container{
+	height: 460px;
+	background-image: url(real-estate-agent-with-house-model-keys.jpg);
+	background-position: center;
+	background-size:cover;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+#main_container{
+	padding: 6%;
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.main_container_header{
+	padding-bottom: 20px;
+	color: #0f4b5e;
+	font-size: 48px;
+}
+
+
+
+.main_header_content{
+	color: #071144;
+}
+
+.main_container_list{
+	list-style: none;
+	padding-bottom: 20px;
+	padding-top: 20px;
+	padding-left: 120px;
+	display: flex;
+	text-align: center;
+
+	
+}
+
+.main_container_list_botton{
+	border: none;
+	background: unset;
+	font-size: 16px;
+	padding-left: 20px;
+	
+
+}
+
+.main_container_list_botton:hover{
+	background: white;
+	font-size: 17px;
+	font-weight: bold;
+	color: red;
+}
```

Comment:
here, add a property to change the mouse shape to a pointer when hover on the button : 
 
```suggestion
.main_container_list_botton:hover{
	background: white;
	font-size: 17px;
	font-weight: bold;
	color: #4fb8d5;
	cursor: pointer;

}
```